packages/runtime/src/useCases/transport/relationshipTemplates/GetRelationshipTemplates.ts

```diff
@@ -51,6 +55,23 @@ export class GetRelationshipTemplatesUseCase extends UseCase<GetRelationshipTemp
                 (t) => t.maxNumberOfAllocations
             )}`,
             [nameof<RelationshipTemplateDTO>((r) => r.forIdentity)]: `${nameof<RelationshipTemplate>((r) => r.cache)}.${nameof<CachedRelationshipTemplate>((t) => t.forIdentity)}`
+        },
+        custom: {
+            [`${nameof<RelationshipTemplateDTO>((r) => r.passwordProtection)}.password`]: (query: any, input: string) => {
+                query[`${nameof<RelationshipTemplate>((t) => t.passwordProtection)}.${nameof<PasswordProtection>((t) => t.password)}`] = input;
+            },
```

Comment:
this should just be an alias

Reply:
an alias would turn a PIN "1234" into a number 1234, so this was my workaround

Reply:
I see, I'm ok with that :)